osu.Game/Screens/Play/Player.cs

```diff
@@ -472,6 +472,16 @@ protected virtual ScoreInfo CreateScore()
             else
                 score.User = api.LocalUser.Value;
 
+            try
+            {
+                var performanceCalculator = ruleset?.CreatePerformanceCalculator(Beatmap.Value, score);
```

Comment:
```suggestion
                var performanceCalculator = ruleset.CreatePerformanceCalculator(Beatmap.Value, score);
```

Reply:
99% confident the above suggestion [will crash for custom rulesets](https://github.com/ppy/osu/blob/ad10a7f0b281f92aed90b614192d62fb05c0c656/osu.Game/Rulesets/Ruleset.cs#L156).

Reply:
Read again, this is about access to ruleset instance.

Reply:
yep this null check is useless. nice catch.